src/store/consts/utils.ts

```diff
@@ -1188,3 +1188,13 @@ export const solToPriorityFee = (sol: number) => {
 }
 
 export const createLoaderKey = () => (new Date().getMilliseconds() + Math.random()).toString()
+
+export const getMainnetCommonTokens = async (): Promise<PublicKey[]> => {
+  const { data } = await axios.get('https://tokens.jup.ag/tokens?tags=verified')
```

Comment:
have you checked if jup handles the `limit` in the query (to fetch only 8 tokens)?

Reply:
yes, i have checked. there is no option to limit number of tokens in query. here is link to API  https://station.jup.ag/docs/token-list/token-list-api